### PR TITLE
Share git existence status

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Common/Git/GitUtility.cs [hunk 1]
```diff
@@ -55,45 +57,15 @@ public static GitDetail TryGetFileDetail(string filePath)
 
         public static GitDetail GetFileDetail(string filePath)
         {
-            if (string.IsNullOrEmpty(filePath))
+            if (string.IsNullOrEmpty(filePath) || !ExistGitCommand())
```
Comment (qinezh) [inline src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: >ExistGitCommand [](start = 51, length = 15)

Do we need leave a message to tell user reason why the git information doesn't be generated?

File: src/Microsoft.DocAsCode.Common/Git/GitUtility.cs [hunk 2]
```diff
@@ -55,45 +57,15 @@ public static GitDetail TryGetFileDetail(string filePath)
...
             {
                 return null;
             }
 
-            if (!Path.IsPathRooted(filePath))
-            {
-                throw new GitException($"{nameof(filePath)} should be an absolute path");
-            }
+            var path = Path.Combine(EnvironmentContext.BaseDirectory, filePath).ToNormalizedPath();
```
Comment (qinezh) [inline src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: As a generic method, it would be better to avoid using EnvironmentContext from Plugins?

Comment (vicancy) [reply src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: It's Environment.CurrentDirectory as default value, I think it is general enough

File: src/Microsoft.DocAsCode.Common/Git/GitUtility.cs (hunk 1 above)
Comment (vicancy) [reply src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: Yes, https://github.com/dotnet/docfx/pull/2086 does

**Pull request merged**